install.php

```diff
@@ -133,7 +133,7 @@ function ArraySearchRecursive($Needle,$Haystack,$NeedleKey="",$Strict=false,$Pat
 	$usePeople=($result->rowCount()>0 && !ArraySearchRecursive('fac_People',$test))?($result->rowCount()>0 && !ArraySearchRecursive('fac_people',$test))?false:true:true;
 
 	// New install so create a user
-	require_once("customers.inc.php");
+	require_once("classes/People.class.php");
```

Comment:
I guess what I'm trying to ask is: `require_once("classes/People.class.php");` will work fine on both installs _and_ upgrades whereas `require_once("customers.inc.php");` will exit on new installs, right? And line 141 `require_once("installer.userconversion.inc.php");` will take care of loading the User class if it is an older install (again regardless of new or update.)

Also I may not have made it clear that customers.inc.php doesn't exist anymore, so if that line needs to stay, there should also be a check for its existence.